### PR TITLE
Suggestion mechanics carson

### LINKED ISSUE #36: Implement suggestion mechanics

### PR DESCRIPTION
**Pull request by carsonlloyd**
Drafting pull request

Closes #36 

*Need to do DISPROVE mechanics
*Resolve TODOs and questions
*Requires updates after #9

### CODE REVIEW

**Comment by carsonlloyd** [comment]
I am still adding disprove logic

**Comment by carsonlloyd** [comment]
This is very dense... need reviewer to walk through logic from 'player_move' all the way through the end of the game ('accusation_made')

File: Board.py [hunk 1]
```diff
@@ -127,8 +129,20 @@ def movePlayer(self, player, action):
...
\ No newline at end of file
+            return False
+
+    def getWeapons(self):
+        '''
+        This function just returns a list of all weapons held by the Board instance.
+        '''
+        return self.weapons
+
+    def getRooms(self)
```
Comment (AmmonHepworth) [inline Board.py]: Needs colon

File: main.py [hunk 2]
```diff
@@ -215,23 +219,97 @@ def parseMessage(jsonMessage):
...
-    	else:
-    		Message.send_false_accusation('ALL_CLIENTS', client, suspect, weapon, room) # should I change this to a sendAll() call?
-    		# go on to display message to clients
+        global correctSuspect, correctWeapon, correctRoom
+        client = message['client_id']
+        suspect = message['suspect']
+        weapon = message['weapon']
+        room = message['room']
+        # confirm accusation is correct
+        case_file = Cards.getCaseFile()
```
Comment (AmmonHepworth) [inline main.py:305]: mixed indentation styles

File: main.py [hunk 3]
```diff
@@ -215,23 +219,97 @@ def parseMessage(jsonMessage):
...
+        if suspect == case_file['suspect'] and weapon == case_file['weapon'] and room == case_file['room']:
+            Message.send_game_win_accusation('ALL_CLIENTS', client, suspect, weapon, room) # should I change this to a sendAll() call?
```
Comment (AmmonHepworth) [inline main.py:307]: sendAll() is more consistent with the code here

File: main.py [hunk 4]
```diff
@@ -215,23 +219,97 @@ def parseMessage(jsonMessage):
...
+            game_won = True
+            # go on to display message to clients
+        else:
+            Message.send_false_accusation('ALL_CLIENTS', client, suspect, weapon, room) # should I change this to a sendAll() call?
```
Comment (AmmonHepworth) [inline main.py:311]: here as well

**Comment by stsnyde** [comment]
Indention issue line 88 in main

**Pull request merged**